src/main/java/io/appium/java_client/MobileCommand.java

```diff
@@ -151,7 +156,10 @@
             postC("/session/:sessionId/appium/performanceData/types"));
         commandRepository.put(GET_PERFORMANCE_DATA,
             postC("/session/:sessionId/appium/getPerformanceData"));
-
+        commandRepository.put(START_RECORDING_SCREEN,
+                postC("/session/:sessionId/appium/startRecordingScreen"));
```

Comment:
@heeseon Didn't get you? It was changed in server side as per standards or convention appium follows.